### ISSUE TITLE
Fixing WRONG_NEWLINES rule with diktatFix does not set correct idents when  WRONG_INDENTATION rule is disabled

### ISSUE THREAD

**Issue by WebTiger89** [saveourtool/diktat#1323]
I had to disable WRONG_INDENTATION rule because it still behaves unexpectedly. Please reopen https://github.com/saveourtool/diktat/issues/1312

I can imagine that it is correct behavior that auto correct WRONG_NEWLINES rule does not set the indents correctly (2 instead of 4) when WRONG_INDENTATION is disabled. Is there a correlation?

But anyway, there is still a strange behavior with WRONG_INDENTATION. For example Ktlint online does not report any WRONG_INDENTATION warnings while it does locally. Also checking the fix checkbox does fix the WRONG_NEWLINES properly, even with my config uploaded. But I'm pretty sure Ktlint online is buggy and does not respect the uploaded config file. It reports MAGIC_NUMBER, although I have disabled it in my config file.

When I enable WRONG_INDENTATION the indents are set properly for WRONG_NEWLINES when running diktatFix but then wrong indents are set at other places in the codebase. For example:

```
Box(
	Modifier
		.align(Alignment.CenterVertically)
		.widthIn(min = 16.dp + 40.dp),
	contentAlignment = Alignment.CenterStart,
)
```
becomes
```
Box(
	Modifier
			.align(Alignment.CenterVertically)
			.widthIn(min = 16.dp + 40.dp),
	contentAlignment = Alignment.CenterStart,
)
```

**Comment by akuleshov7** [comment]
@0x6675636b796f75676974687562 pls investigate as you are JB guy 😄

**Comment by 0x6675636b796f75676974687562** [comment]
@akuleshov7, the lack of _continuation indent_ in your 2nd example (despite `extendedIndentAfterOperators` is **on**) is exactly #1330.

IDEA's code style flags are:

 * `CONTINUATION_INDENT_FOR_EXPRESSION_BODIES` and
 * `CONTINUATION_INDENT_IN_IF_CONDITIONS`,

both are **on** by default as per the Kotlin code style guidelines.
